docs/user/debugging.md

```diff
@@ -4,154 +4,243 @@
+    all files associated with a given task, including the task logs, you can
+    run `turbinia-client result task <task id>` and it will download a
+    compressed file with all of the output.
+*   **Task dependency execution logs**: These are the logs that are generated by
+    binaries executed from a Task. For example, Plaso generates its own log
+    file, and that gets saved by the Task. As with the Task logs, these are
+    stored in the Task output directory.  See the above `Task logs` section for
+    how to determine the paths and retrieve the output.
+*   **Google Cloud Error Reporting / Stackdriver Logs**: If you have deployed
+    Turbinia in GKE and/or have `STACKDRIVER_TRACEBACK` enabled in the config,
+    the logs and/or traceback exceptions will also be logged to GCP Monitoring
+    for the same project that Turbinia is in.
 
 ## Request Debugging
 
 ### Finding previous request data
-By default `turbiniactl` will make a processing request and immediately return.
-It will print out the request ID when it makes the request, and you can use this to
-see the current status of the request with `turbiniactl status -r <request id>`.  If
-you specify the `-w` flag (`turbiniactl -w status -r <request id>`), the client will
-wait for all Tasks to complete and then return.  Killing the client will not affect
-the processing, and you can resume waiting for the task results with the same command.
-
-If you do not have your request ID, you can list all recent requests like this:
-```
-$ turbiniactl status -i
+When processing requests are submitted via the client (e.g.
+`turbinia-client submit <evidence type>`) it will print out the request ID for
+that request. This can be used with
+`turbinia-client status request <request id>` to get the current state of the
+processing request.  Alternately you can also use the [web
+ui](turbinia-web-ui.md) to get details as well.  For more details on using the
+client [see the documentation here](turbinia-client.md).  If you do not have
+your request ID, you can list a summary of all recent requests and their request
+IDs with: `turbinia-client status summary`.
 
-# Turbinia report for Requests made within 7 days
-* 2 requests were made within this timeframe.
 
-## Request ID: 5198949e3fdb8dk569de1268000f97d8
-* Last Update: 2021-02-10T08:39:27.446000Z
-* Requester: turbiniauser
-* Task Count: 441
+## Task Debugging
 
-## Request ID: 69de1268000f97d85198949e3fdb8dk5
-* Last Update: 2021-01-12T07:09:03.123000Z
-* Requester: turbiniauser
-* Task Count: 43
-```
+### Full Task report data
+To see the request output you can specify
+`turbinia-client status request <request id>`. By default this output will be
+filtered to only show high priority Task report output (as determined by the
+Task at runtime) in order to filter out uninteresting report info. You can
+specify `-p <prio num>` to set what priority you want to show in the output
```

Comment:
Should we add something about how priorities work for users to know what number to use to filter? e.g. There are three categories of priorities Turbinia uses. These include, Medium >= 50, High <= 20, Critical <= 10.

Reply:
I moved up the parentheses from below per your suggestion and updated it a bit to match here.